src/hooks/screen/useRootChannelListHook.ts

```diff
@@ -1,7 +1,18 @@
 import * as React from 'react';
+import PushNotification from 'react-native-push-notification';
+import messaging from '@react-native-firebase/messaging';
+import {Alert, PushNotificationPermissions} from 'react-native';
+import {openSettings} from 'react-native-permissions';
 
 import ChannelList from '../../database/schema/ChannelListSchema';
+import StorageUtils from '../../utils/storage';
 import useLocalDatabaseHook from '../../database/hooks/useLocalDatabaseHook';
+import {
+  PERMISSION_STATUS_ACCEPTED,
+  PERMISSION_STATUS_BLOCKED,
+  PERMISSION_STATUS_PENDING
+} from '../../utils/constants';
+import {fcmTokenService} from '../../service/users';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new imports introduced in this hunk are not being used anywhere in the provided code. If they are not used elsewhere in the file, it would be best to remove them to avoid unnecessary imports which can lead to slower load times and potential confusion for future developers.
```diff
- import PushNotification from 'react-native-push-notification';
- import messaging from '@react-native-firebase/messaging';
- import {Alert, PushNotificationPermissions} from 'react-native';
- import {openSettings} from 'react-native-permissions';
- import StorageUtils from '../../utils/storage';
- import {
-   PERMISSION_STATUS_ACCEPTED,
-   PERMISSION_STATUS_BLOCKED,
-   PERMISSION_STATUS_PENDING
- } from '../../utils/constants';
- import {fcmTokenService} from '../../service/users';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->